src/main/java/com/powsybl/openloadflow/network/impl/PropagatedContingency.java

```diff
@@ -108,7 +109,7 @@ private static PowerShift getLoadPowerShift(Load load, boolean slackDistribution
         List<PropagatedContingency> propagatedContingencies = new ArrayList<>();
         for (int index = 0; index < contingencies.size(); index++) {
             Contingency contingency = contingencies.get(index);
-            PropagatedContingency propagatedContingency = PropagatedContingency.create(network, contingency, index, false, false, slackDistributionOnConformLoad);
+            PropagatedContingency propagatedContingency = PropagatedContingency.create(network, contingency, index, false, false, slackDistributionOnConformLoad, false);
```

Comment:
For me: for an HVDC line in AC emulation, in AC sensitivity analysis and in AC security analysis, we have to:
- Handle it;
- Throw an exception (it is what we do for AC security analysis);
- Not do what is done for AC sensitivity analysis: saying nothing and try to handle it as set point.